Library/Homebrew/cmd/orphaned.rb

```diff
@@ -0,0 +1,13 @@
+#:  * `orphaned`:
+#:    Lists kegs that were installed as dependencies of other kegs, but are no
+#:    longer required for any explicitly-installed keg.
+
+require "keg"
+
+module Homebrew
+  module_function
+
+  def orphaned
+    puts Keg.orphaned.map { |k| "#{k.name} #{k.version}" }
```

Comment:
Agreed; I think orphaned should be version agnostic and probably actually never print versions at all.